playbooks/roles/nginx/templates/edx/app/nginx/sites-available/cms.j2

```diff
@@ -100,6 +100,10 @@ error_page {{ k }} {{ v }};
     {{ NGINX_EDXAPP_CMS_APP_EXTRA }}
   }
 
+  {% if NGINX_EDXAPP_CMS_EXTRA|length %}
```

Comment:
@xirdneh, wouldn't it be better to check if the variable is defined and check is truthiness since `""` is false.

Reply:
@lgp171188 Updated the PR.